src/platform/intel/cavs/lib/clk.c

```diff
@@ -102,15 +102,27 @@ void platform_clock_on_wakeup(void)
 }
 #endif
 
-void platform_clock_on_waiti(void)
+void platform_clock_waiti_entry(void)
```

Comment:
Note that the old function is being fixed by #3113 on master branch. Is there any rationale behind changing the name while this new piece of deferred clock setup is added?

Reply:
I didn't notice what #3113 is changing, the reason I am changing the naming is just to make the _xx_entry() and _xx_exit() symmetric and are invoked in the platform_wait_for_interrupt() together.